NExtends/Attributes/CulturedDescriptionAttribute.cs

```diff
@@ -34,7 +29,7 @@ public static string GetName(Enum enumValue)
 			var type = enumValue.GetType();
 			var memInfo = type.GetMember(enumValue.ToString());
 			var attributes = memInfo[0].GetCustomAttributes(typeof(CulturedDescriptionAttribute), false);
-			return (attributes.Length > 0) ? ((CulturedDescriptionAttribute)attributes[0]).Description : String.Empty;
+			return (attributes.Count() > 0) ? ((CulturedDescriptionAttribute)attributes.ElementAt(0)).Description : String.Empty;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Use '.Any()' to test whether this 'IEnumerable<System.Attribute>' is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=csharpsquid%3AS1155)